packages/server/src/channels/service.ts

```diff
@@ -1,23 +1,23 @@
 import { uuid } from '@botpress/messaging-base'
+import {
+  Channel,
+  MessengerChannel,
+  SlackChannel,
+  SmoochChannel,
+  TeamsChannel,
+  TelegramChannel,
+  TwilioChannel,
+  VonageChannel
+} from '@botpress/messaging-channels'
 import { Service, DatabaseService } from '@botpress/messaging-engine'
-import { Channel } from './base/channel'
-import { ConduitInstance } from './base/conduit'
-import { DiscordChannel } from './discord/channel'
-import { MessengerChannel } from './messenger/channel'
-import { SlackChannel } from './slack/channel'
-import { SmoochChannel } from './smooch/channel'
 import { ChannelTable } from './table'
-import { TeamsChannel } from './teams/channel'
-import { TelegramChannel } from './telegram/channel'
-import { TwilioChannel } from './twilio/channel'
-import { VonageChannel } from './vonage/channel'
 
 export class ChannelService extends Service {
   private table: ChannelTable
 
-  private channels: Channel<ConduitInstance<any, any>>[]
-  private channelsByName: { [name: string]: Channel<ConduitInstance<any, any>> }
-  private channelsById: { [id: string]: Channel<ConduitInstance<any, any>> }
+  private channels: Channel<any, any, any, any>[]
+  private channelsByName: { [name: string]: Channel<any, any, any, any> }
+  private channelsById: { [id: string]: Channel<any, any, any, any> }
```

Comment:
That's a lot of any!

Reply:
Fixed this by adding a ChannelTemplate class that has all the generics and a Channel interface that just has parameters of type any